src/Content/RichContent.js

```diff
@@ -0,0 +1,31 @@
+import { createElement, Component } from 'react';
+import { object, string } from 'prop-types';
+
+export default class RichContent extends Component {
+    static propTypes = {
+        sanitizedRawHtml: string.isRequired,
+        wrapperTag: string,
+        wrapperProps: object
+    };
+
+    static defaultProps = {
+        wrapperTag: 'span',
+        wrapperProps: {
+            className: 'peregrine-raw-html'
+        }
+    };
+
+    render() {
+        // JSX tags must be in PascalCase
```

Comment:
FWIW, this is only accurate for bare identifiers. Member expressions do not have this limitation.

![image](https://user-images.githubusercontent.com/5233399/41299903-1fc67554-6e2a-11e8-94c5-2878021543e0.png)